js/app/src/Index.svelte

```diff
@@ -255,6 +255,9 @@
 			status_callback: handle_status
 		});
 		config = app.config;
+		if (window.location.protocol === "https:") {
```

Comment:
How come we need to do this check if the client already does it?

Reply:
Actually we might not need this one. I added it here first and then to the client, I'll check with this removed as well.

Reply:
Tested and it seems to work, thanks @freddyaboulton!